views/pages/login.ejs

```diff
@@ -1,17 +1,31 @@
 <!doctype html>
 <html lang="en">
-<% include ../includes/head.ejs %>
+<head>
+    <% include ../includes/head.ejs %>
+    <link href="/css/login.css" rel="stylesheet" type="text/css">
+</head>
 <body>
 <% include ../includes/header.ejs %>
 <section class="section is-medium">
     <div class="container has-text-centered">
+        <p class="accroche">Vous pouvez dès à présent tester les identités de niveau faible (eidas1), substantiel (eidas2) et fort (eidas3) sur l'environnement de qualification. En production, vous n'aurez cependant accès pour l'instant qu'à des identités de niveau faible (eidas1).</p>
```

Comment:
This example is used to demonstrate the basic flow for FC connection.
It is a good idea to use it to demonstrate also the eidas flow but we should keep simplicity and clarity in mind.

I suggest you move this to a dedicated documentation on eidas and simply add a 'qu'est ce que c'est ?' link